config/locales/devise/en.yml

```diff
@@ -5,22 +5,22 @@ en:
       already_confirmed: Your email address has already been confirmed. %{action}
       confirmed: You have confirmed your email address
       confirmed_but_must_set_password: You have confirmed your email address
-      send_instructions: >
-        You will receive an email with instructions for how to confirm your
-        email address in a few minutes.
-      send_paranoid_instructions: >
-        You will receive an email with instructions for how to confirm your email
-        address in a few minutes.
+      send_instructions: 'You will receive an email with instructions for how to confirm your email address in a few minutes.
+
+'
```

Comment:
Is there a way for `i18n-tasks normalize` to ignore wrapping with `>`? Do we prefer `>` or string quotes? If string quotes, I would vote to strip them because it looks weird and messy. I like the cleanliness of `>` and being able to wrap at a comfortable reading line length.